code/__DEFINES/time.dm

```diff
@@ -18,7 +18,6 @@
 #define VALENTINES				"Valentine's Day"
 #define APRIL_FOOLS				"April Fool's Day"
 #define EASTER					"Easter"
```

Comment:
```suggestion
#define EASTER					"Easter"
#define PRIDE_MONTH				"Pride Month"
```
Ive noticed that you misstaken removed the define for Pride Month!
Its fine misstakes happen.

Reply:
Not funny. Didn't laugh.

Reply:
This will also make it compile!